src/systems/filecoin_blockchain/storage_power_consensus/_index.md

```diff
@@ -33,6 +33,16 @@ We must distinguish between both types of "miners" (storage and block miners). {
 
 However, given Filecoin's "useful Proof-of-Work" is achieved through file storage (PoRep and PoSt), there is little overhead cost for storage miners to participate in leader election. Such a {{<sref storage_miner_actor>}} need only register with the {{<sref storage_power_actor>}} in order to participate in Expected Consensus and mine blocks.
 
+{{<label storage_power>}}
+## On Power
+
+Claimed power is assigned to every sector as a static function of its `SectorStorageWeightDesc` which includes `SectorSize`, `Duration`, and `DealWeight`. DealWeight is a measure that maps size and duration of active deals in a sector during its lifetime to its impact on power and reward distribution. A CommittedCapacity Sector (see {{<sref storage_mining_subsystem "Sector Types">}}) will have a DealWeight of zero but all sectors have an explicit Duration which is defined from the ChainEpoch that the sector comes online in a ProveCommit message to the Expiration ChainEpoch of the sector. In principle, power is the number of votes a miner has in leader election and it is a point in time concept of storage. However, the exact function that maps `SectorStorageWeightDesc` to claimed `StoragePower` and `BlockReward` will be announced soon.
+
+More precisely,
+
+- Nominal power = claimed power, unless the miner is in DetectedFault state. Nominal power is used to determine total network storage power for purposes of consensus minimum.
+- Consensus power = nominal power, unless the miner fails to meet consensus minimum, or is undercollateralized.
```

Comment:
This is great

Reply:
nit: could add `claimed power (to add how declared faults factor in)` above but looks great.

Reply:
thanks and it has been updated.